package/efs/rancher-efs

```diff
@@ -106,6 +106,15 @@ create() {
 
     get_meta_data
 
+    # Check if an EFS volume with the same creation token exists
+    local existing_fs
+    existing_fs=`aws efs describe-file-systems --region ${EC2_REGION} --creation-token ${OPTS[name]} | grep -c ${OPTS[name]}`
+    
```

Comment:
What's the purpose of doing so? --creation-token should already ensure idempotent creation.